integrations/webchat/examples/extension-with-custom-response/DOCS.md

```diff
@@ -0,0 +1,29 @@
+To display information from an extension in a custom response, follow these steps.
+
+1. Set up the extension using these instructions: [Building a custom extensions](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-build-custom-extension)
+2. Assign the data from the extension to a session variable. You can find documentation for this at [Accessing extension response data](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-call-extension#extension-access-response)
+3. Add a step to your action that returns a `user_defined` response. Use the JSON editor to send the following response.
+```json
+{
+  "generic": [
+    {
+      "response_type": "user_defined"
```

Comment:
Should we just go ahead and use the `user_defined.template_name` pattern here rather than make them figure it out when they want to add a second `user_defined` response type?

Reply:
That seems like perhaps something to include if we're going to have an actual custom response demo.

And FYI, I had Abe use something like `action_type` instead of `template_name` in some of the work he was doing. `template_name` doesn't seem quite as  obvious to me what it's for.

Reply:
`action` is an overloaded word in WA. Is just `name` or `type` enough?

Reply:
Sure